azkaban-db/src/main/java/azkaban/db/DatabaseSetup.java

```diff
@@ -23,59 +23,35 @@
 import java.io.IOException;
 import java.sql.Connection;
 import java.sql.SQLException;
-import java.util.HashMap;
 import java.util.HashSet;
-import java.util.Map;
 import java.util.Set;
 import org.apache.commons.dbutils.QueryRunner;
 import org.apache.commons.io.IOUtils;
 import org.apache.log4j.Logger;
 
-/**
- * TODO kunkun-tang: this class is quite messy now, needs to be refactored.
- */
 public class DatabaseSetup {
```

Comment:
Could you add a javadoc explaining what this class does? According to the style guide, every public interfaces need to have javadocs.

Reply:
Thanks. Just added.